simapp/app.yaml

```diff
@@ -63,3 +63,8 @@ modules:
   - name: staking
     config:
       "@type": cosmos.staking.module.v1.Module
+
+  - name: group
+    config:
+      "@type": cosmos.group.v1.module.v1.Module
+      max_metadata_len: 255
```

Comment:
what about max_execution_period?

Reply:
Added default value for `max_execution_period` in `x/group/module/module.go`'s `provideModule` func, since it has `time` package dependency

Reply:
On a note, now added `max_execution_period` to app.yaml with value in seconds.
I think adding it in group's `module.go` just makes it more complex.

Reply:
isn't there a way to encode that here?